plugin/hpsa/hpsa.py

```diff
@@ -231,6 +231,17 @@ def _disk_status_of(hp_disk, flag_free):
 
     return disk_status
 
+def _disk_link_type_of(hp_disk):
+    disk_interface = hp_disk['Interface Type']
+    if disk_interface == 'SATA':
+        return Disk.LINK_TYPE_ATA
+    elif disk_interface == 'Solid State SATA':
+        return Disk.LINK_TYPE_ATA
+    elif disk_interface == 'SAS':
+        return Disk.LINK_TYPE_SAS
+
```

Comment:
I'll check it for you, @cathay4t. Didn't have a chance to get my hands on a SAS SSD today. :/